src/components/DefaultRedirectionSnackbar/helpers.js

```diff
@@ -0,0 +1,90 @@
+import { deconstructRedirectLink, hasQueryBeenLoaded } from 'cozy-client'
+
+export const NAV_APP_SHOW_THRESHOLD = 3
+
+export const HOME_DEFAULT_REDIRECTION = 'home/'
+
+export const incrementNavAppShowCount = async (
+  client,
+  instanceSettingsResult
+) => {
+  const { nav_app_show_count } = instanceSettingsResult.data.attributes
+
+  const instanceSettings = {
+    data: {
+      ...instanceSettingsResult.data,
+      attributes: {
+        ...instanceSettingsResult.data.attributes,
+        nav_app_show_count: nav_app_show_count ? nav_app_show_count + 1 : 1
+      }
+    }
+  }
+
+  return await client.save(instanceSettings)
+}
+
+export const disableDefaultRedirectionSnackbar = async (
+  client,
+  instanceSettingsResult
+) => {
+  const instanceSettings = {
+    data: {
+      ...instanceSettingsResult.data,
+      attributes: {
+        ...instanceSettingsResult.data.attributes,
+        default_redirection_snackbar_disabled: true
+      }
+    }
+  }
+
+  return await client.save(instanceSettings)
+}
+
+export const setDefaultRedirectionToHome = async (
+  client,
+  instanceSettingsResult
+) => {
+  const instanceSettings = {
+    data: {
+      ...instanceSettingsResult.data,
+      attributes: {
+        ...instanceSettingsResult.data.attributes,
+        default_redirection: HOME_DEFAULT_REDIRECTION
```

Comment:
this one, OK ;). Can you update this documentation https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.settings.md#iocozysettings ?